webapp/app.py

```diff
@@ -0,0 +1,21 @@
+from canonicalwebteam.blog import BlogViews
+from canonicalwebteam.blog.flask import build_blueprint
+from canonicalwebteam.flask_base.app import FlaskBase
+from canonicalwebteam.templatefinder import TemplateFinder
+
+app = FlaskBase(
+    __name__,
+    "cn.ubuntu.com",
+    template_folder="../templates",
+    static_folder="../static",
+    template_404="templates/404.html",
```

Comment:
Aaaah the `404.html` is inside a folder called `templates/templates`? Yuck yuck yuck.

I think at some point we should start to set some standards around this stuff. Personally I like the idea of following a Jekyll like naming convention - `templates/_includes` and `templates/_layouts`. But I prefer `404.html` etc. to be at the top level `templates` directory.

I've added an item to the next dev ketchup to talk about the naming conventions. If you wouldn't mind moving `404.html` and `500.html` up one level that would be great.

Reply:
Agree. This should be changed. I moved the 404 and 500 for now.
We can make "bigger" changes after when the scope of the pr is resolved :)